trace/opencensus/trace/ext/mysql/trace.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+import logging
+import mysql.connector
+
+import wrapt
+
+from opencensus.trace import thread_local
+
+log = logging.getLogger(__name__)
+
+MODULE_NAME = 'mysql'
+
+
+def trace():
+    """Wrap the mysql connector to trace it."""
+    log.info('Integrated module: {}'.format(MODULE_NAME))
+    conn_func = mysql.connector.connect
+    conn_module = inspect.getmodule(conn_func)
+    wrapped = wrap_conn(conn_func)
+    setattr(conn_module, conn_func.__name__, wrapped)
+
+
+def wrap_conn(conn_func):
```

Comment:
@jonparrott Do you have any particular opinions on wrapping, e.g., MySQL or other third-party objects with wrappers that call tracing functions?